libkbfs/disk_block_cache.go

```diff
@@ -138,17 +181,25 @@ func newDiskBlockCacheStandardFromStorage(config diskBlockCacheConfig,
 	startedCh := make(chan struct{})
 	startErrCh := make(chan struct{})
 	cache = &DiskBlockCacheStandard{
-		config:     config,
-		maxBlockID: maxBlockID.Bytes(),
-		tlfCounts:  map[tlf.ID]int{},
-		tlfSizes:   map[tlf.ID]uint64{},
-		log:        log,
-		blockDb:    blockDb,
-		metaDb:     metaDb,
-		tlfDb:      tlfDb,
-		startedCh:  startedCh,
-		startErrCh: startErrCh,
-		shutdownCh: make(chan struct{}),
+		config:           config,
+		maxBlockID:       maxBlockID.Bytes(),
+		tlfCounts:        map[tlf.ID]int{},
+		tlfSizes:         map[tlf.ID]uint64{},
+		hitMeter:         metrics.NewMeter(),
```

Comment:
When I wrote that code, I avoided creating meters only because if the registry was `nil`, it would be doing work that would never be seen. If we're just using the meter type here, and we're exposing them via `.kbfs_status`, then probably don't need to respect `MetricsRegistry`.